frontend/ui/elements/reader_menu_order.lua

```diff
@@ -31,6 +31,7 @@ local order = {
     },
     navi_settings = {
         "toc_alt_toc",
+        "toc_show_book_title",
```

Comment:
I think it should be below, above toc_items_per_page, as it's about the look/display of the ToC.
Here, toc_alt_toc and toc_ticks_level_ignore are about the ToC content